app/design/frontend/default/models/template/test/test.phtml

```diff
@@ -0,0 +1,188 @@
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+<br />
+<div id="main-div">
+<div id="left-div">
+<h3 id="heading"><center>Enter all fields to Add data</center></h3>
+<form role="form" action="<?php echo $this->getUrl('test/user_view/save'); ?>" method="post">
+ <div id="form_div">
+  <label id="test_id_label" style="display:none;">Test_Id</label>
+  <input id="test_id" type="text" name="testid" style="display:none;" />
+  <br /><br />
+  <label>Title</label>
+  <input id="title" type="text" name="title" />
+  <br /><br />
+  <label>File Name</label>
+  <input id="file_name" type="text" name="filename" />
+  <br /><br />
+  <label>Content</label>
+  <input id="content" type="text" name="content" />
+  <br /><br />
+  <label>Status</label>
+  <input id="status" type="text" name="status" />
+  <br /><br />
+  <input id="add_button" type="submit" value="Add" disabled="disabled" />
+  <input type="button" class="edit" value="Edit" style="cursor:pointer;" />
```

Comment:
make html proper as per magento theme. if i open this page and view html, it should look part of theme. not plan simple html